cmd/proxy/actions/app.go

```diff
@@ -41,7 +41,11 @@ func App(conf *config.Config) (*buffalo.App, error) {
 
 	// mount .netrc to home dir
 	// to have access to private repos.
-	initializeNETRC(conf.Proxy.NETRCPath)
+	initializeAuthFile(conf.Proxy.NETRCPath)
```

Comment:
This is like a super tiny nit:

you can probably do 

`initializeAuthFile(conf.Proxy.NetRCPath, conf.HGRCPath)`

and change the function signature to 

`func initializeAuthFile(files []string)`

Reply:
I think it's fine as it is, the loop inside initializeAuthFile is no needed for now when we have two files, maybe in future when there will be more file initialized we could think of that